xml/deploy_cephadm.xml

```diff
@@ -888,10 +847,20 @@ config: OK
   </sect2>
 
   <sect2 xml:id="deploy-cephadm-deploy">
-   <title>Deploy Cluster</title>
+   <title>Update Nodes and Deploy Cluster</title>
    <para>
-    Deploy the previously configured minimal &ceph; cluster by running the
-    following command:
+    Before you deploy the cluster, update all software packages on all nodes:
+   </para>
+<screen>&prompt.smaster;ceph-salt update</screen>
+   <tip>
+    <para>
+     To reboot the nodes after they are updated, append the
+     <option>--reboot</option> option:
+    </para>
+<screen>&prompt.smaster;ceph-salt update --reboot</screen>
+   </tip>
+   <para>
+    After the nodes are updated, deploy the minimal cluster:
```

Comment:
What about adding a tip, explaining that salt-master is never rebooted automatically?

If salt-master needs to be rebooted, the reboot should be done manually:

![Screenshot from 2020-08-31 12-10-02](https://user-images.githubusercontent.com/14297426/91821973-0b92a780-ec2f-11ea-964b-ecca6acde6cf.png)

![Screenshot from 2020-08-31 11-50-09](https://user-images.githubusercontent.com/14297426/91821999-151c0f80-ec2f-11ea-990c-250c5403d6d6.png)